content/departments/engineering/dev/admin-exp/devops/managed/v1.1/mi1-1_enable_executors_process.md

```diff
@@ -121,3 +99,42 @@ Link Monitoring email notification channels we created from the terraform module
 ![create-budget-04](https://storage.googleapis.com/sourcegraph-assets/create-executor-budgets-04.png)
 
 ![create-budget-02](https://storage.googleapis.com/sourcegraph-assets/create-executor-budgets-02.png)
+
+## Troubleshooting
+
+Ensure instance groups are there
+
+```sh
+$ gcloud compute instance-groups list --zones=$zone --project=$PROJECT
+NAME                             LOCATION       SCOPE  NETWORK                MANAGED  INSTANCES
+batches--sourcegraph-executor    us-central1-a  zone   sourcegraph-executors  Yes      0
+codeintel--sourcegraph-executor  us-central1-a  zone   sourcegraph-executors  Yes      0
+```
+
+Ensure `minNumReplicas` is greater than `0`
+
+```sh
+$ gcloud compute instance-groups managed describe batches--sourcegraph-executor --zone=$ZONE --project=$PROJECT --format=json | jq '.autoscaler.autoscalingPolicy'
+{
+  "minNumReplicas": 1,
+}
+```
+
+```sh
+$ gcloud compute instance-groups managed describe codeintel--sourcegraph-executor --zone=$ZONE --project=$PROJECT --format=json | jq '.autoscaler.autoscalingPolicy'
+{
+  "minNumReplicas": 1,
+}
+```
+
+Ensure there is an active instance belong to one of the instance group (notes `batches--sourcegraph-executor-rqfs`)
```

Comment:
```suggestion
Ensure there is an active instance belong to one of the instance group (notes the `batches--sourcegraph-executor-rqfs` instance). Sometimes it may take GCP longer to spawn a new instance, so be patient. If no new instance is created for an unreasonable amount of time, consult GCP documentation for next step.
```